core/process/aru.php

```diff
@@ -706,9 +706,21 @@
 		header('Content-Type: application/json');
 
 		echo $json;
-
 		break;
 
+	case 'pokemon_coordinates_area':
+		$json="";
+		$req 		 = "SELECT MAX(latitude) AS max_latitude, MIN(latitude) AS min_latitude, MAX(longitude) AS max_longitude, MIN(longitude) as min_longitude FROM pokemon";
```

Comment:
pls query spawnpoint or scannedlocation table. This will be way faster (see my other comment).